synapse/cortex.py

```diff
@@ -5501,6 +5501,8 @@ async def _pushBulkEdits(self, layr0, layr1, pdef):
         csize = pdef.get('chunk:size', s_const.layer_pdef_csize)
         qsize = pdef.get('queue:size', s_const.layer_pdef_qsize)
         soffs = pdef.get('offs', -1)
+        if soffs <= 0:
+            soffs = -1
```

Comment:
Should we always just be subtracting 1 from the default offset?

Reply:
seems like if it was configured with a non-zero offset we would miss the first edit

Reply:
Just seems inconsistent whether the sync behavior is inclusive of the offset specified